.github/workflows/build-image.yaml

```diff
@@ -17,43 +16,98 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - dockerfile: Dockerfile
+            platform: linux/amd64
+            scope: build-amd
+          - dockerfile: Dockerfile.aarch64
+            platform: linux/arm64
+            scope: build-arm
     steps:
       - name: Check out code
         uses: actions/checkout@v4
-      - name: Add latest tag
-        if: ${{ github.ref_name == env.MAIN_BRANCH_NAME }}
-        run: |
-          echo "IMG_TAGS=${{ env.IMG_TAGS }} latest" >> $GITHUB_ENV
-      - name: Add branch name tag
-        if: ${{ github.ref_name != env.MAIN_BRANCH_NAME }}
-        run: |
-          echo "IMG_TAGS=${{ env.IMG_TAGS }} ${{ github.ref_name }}" >> $GITHUB_ENV
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-      - name: Build Image
-        id: build-image
-        uses: redhat-actions/buildah-build@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ env.IMG_REGISTRY_HOST }}/${{ env.IMG_REGISTRY_ORG }}/limitador
+      - name: Login to container registry
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.IMG_REGISTRY_USERNAME }}
+          password: ${{ secrets.IMG_REGISTRY_TOKEN }}
+          registry: ${{ env.IMG_REGISTRY_HOST }}
+      - name: Build and push by digest
+        id: build
+        uses: docker/build-push-action@v5
         with:
-          image: limitador
-          tags: ${{ env.IMG_TAGS }}
-          platforms: linux/amd64,linux/arm64
-          dockerfiles: |
-            ./Dockerfile
+          context: .
+          labels: ${{ steps.meta.outputs.labels }}
```

Comment:
Which labels are set by default from the output of the meta step?

Reply:
This is the JSON output for this build step (taken from the workflow logs):

```json
{
    "tags": [
      "quay.io/kuadrant/limitador:quicker-workflow"
    ],
    "labels": {
      "org.opencontainers.image.created": "2024-01-08T15:23:11.687Z",
      "org.opencontainers.image.description": "Rate limiter",
      "org.opencontainers.image.licenses": "Apache-2.0",
      "org.opencontainers.image.revision": "a39035119c7899901dfc785e[59](https://github.com/Kuadrant/limitador/actions/runs/7449493828/job/20266248072?pr=222#step:4:68)380d52b9b11986",
      "org.opencontainers.image.source": "https://github.com/Kuadrant/limitador",
      "org.opencontainers.image.title": "limitador",
      "org.opencontainers.image.url": "https://github.com/Kuadrant/limitador",
      "org.opencontainers.image.version": "quicker-workflow"
    },
    "annotations": [
      "manifest:org.opencontainers.image.created=2024-01-08T15:23:11.[68](https://github.com/Kuadrant/limitador/actions/runs/7449493828/job/20266248072?pr=222#step:4:77)7Z",
      "manifest:org.opencontainers.image.description=Rate limiter",
      "manifest:org.opencontainers.image.licenses=Apache-2.0",
      "manifest:org.opencontainers.image.revision=a39035119c7899901dfc785e59380d52b9b11986",
      "manifest:org.opencontainers.image.source=https://github.com/Kuadrant/limitador",
      "manifest:org.opencontainers.image.title=limitador",
      "manifest:org.opencontainers.image.url=https://github.com/Kuadrant/limitador",
      "manifest:org.opencontainers.image.version=quicker-workflow"
    ]
  }
```